From de27881a7a72ce1c99650d96e9de29e358f4fe79 Mon Sep 17 00:00:00 2001
From: Joakim Zhang <qiangqing.zhang@nxp.com>
Date: Wed, 12 Sep 2018 19:06:32 +0800
Subject: [PATCH 4635/5242] MLK-19557-2 can: flexcan: note for clock source
 select

commit  9a4d9636d0dbb4d5cf430dfcdbbf2b974992e501 from
https://source.codeaurora.org/external/imx/linux-imx.git

FLEXCAN has supported selecting the clock source to the CAN Protocol Engine (PE).
It's SoC Implementation dependent. Refer to RM for detailed definition of each SoC.

Signed-off-by: Joakim Zhang <qiangqing.zhang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/net/can/fsl-flexcan.txt    |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/net/can/fsl-flexcan.txt b/Documentation/devicetree/bindings/net/can/fsl-flexcan.txt
index 278644a..18827ce 100644
--- a/Documentation/devicetree/bindings/net/can/fsl-flexcan.txt
+++ b/Documentation/devicetree/bindings/net/can/fsl-flexcan.txt
@@ -34,6 +34,9 @@ Optional properties:
 - trx_stby_gpio : standby gpio
 - trx_nerr_gpio : NERR gpio
 - disable-fd-mode : disable CAN FD mode support. Valid since i.MX8 series.
+- clk-src : Selects the clock source to the CAN Protocol Engine (PE). It's SoC
+	    Implementation dependent. Refer to RM for detailed definition.
+	    0: clock source 0 1: clock source 1
 
 Example:
 
-- 
1.7.9.5

